src/components/Class/index.js

```diff
@@ -291,7 +312,22 @@ function Class({ classToEdit, indicator }) {
 
     for (let [fieldName, value] of formData.entries()) {
       if (value) {
-        if (fieldName === "max_enrolment") {
+        const weekDday = Object.values(day);
+        if (fieldName === "start_time") {
+          let incrementedDate = new Date(value);
+          let newDate;
+          var i = 1;
+          while (i <= 7) {
+            incrementedDate = moment(incrementedDate).add(1, "days")._d;
+            let Day = incrementedDate.toString().split(" ")[0];
+            if (days[weekDday[0]] === Day) {
+              newDate = incrementedDate;
+              break;
+            }
+            i = i + 1;
+          }
+          formFields[fieldName] = moment.utc(newDate).format("YYYY-MM-DD");
```

Comment:
It's not able to resolve `date-fns-tz` even after `[installing](https://www.npmjs.com/package/date-fns-tz)`

Reply:
Not sure about this as I was able to get the imports to work in a CodeSandbox, which handles dependency installation for you, but this may be related (https://github.com/marnusw/date-fns-tz/issues/133).